integrations/hubspot/client.go

```diff
@@ -0,0 +1,71 @@
+package hubspot
+
+import (
+	"context"
+
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkintegrations"
+	"go.autokitteh.dev/autokitteh/sdk/sdkmodule"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type integration struct{ vars sdkservices.Vars }
+
+var (
+	authType      = sdktypes.NewSymbol("authType")
```

Comment:
Apparently, the web UI sets and expects `auth_type` and not `authType` - so we have both in many integrations. No need to keep this error in new integrations.

```suggestion
	authType      = sdktypes.NewSymbol("auth_type")
```